src/NuGet.Core/NuGet.Packaging/RuntimeModel/JsonRuntimeFormat.cs

```diff
@@ -52,8 +52,9 @@ public static void WriteRuntimeGraph(string filePath, RuntimeGraph runtimeGraph)
             using (var writer = new JsonObjectWriter(jsonWriter))
             {
                 jsonWriter.Formatting = Formatting.Indented;
-
+                jsonWriter.WriteStartObject();
```

Comment:
Does this change the content of what's written? Does it add an extra `{}`?

Reply:
Yes, it does. It was broken before because the RuntimeGraph json was malformed (missing `{}`).